.idea/icgc-download-client.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Usually we don't commit configuration files from the IDE/Editor. It may be a good idea to add the `.idea/` directory to the `.gitignore` file. 

https://git-scm.com/docs/gitignore